docs/DeveloperGuide.md

```diff
@@ -461,34 +565,94 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 3a1. SalesNote shows an error message.
 
       Use case resumes at step 2.
+    
+* 5a. No related tasks found.
 
-#### Use case: Add an order
+  Use case ends.
+
+#### Use case: Edit a task
 
 **MSS**
 
-1. User requests to add a specific order to the list
-2. SalesNote adds the order
+1. User requests to list tasks.
+2. SalesNote shows a list of tasks.
+3. User requests to edit the details of a specific task in the list.
+4. SalesNote edits the details of the task.
 
    Use case ends.
 
 **Extensions**
 
-* 1a. The format of the request is invalid.
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The format / details of the request are invalid.
+
+    * 3a1. SalesNote shows an error message.
+    * 3a2. User enters a new request.
+    * Steps 3a1-3a2 are repeated until the request is valid.
+
+      Use case resumes from step 4.
+
+* 4a. The user has not made any changes to the task details.
+
+    Use case resumes at step 3.
+
+* 4b. A task with the edited details already exists in the task list.
 
-    * 1a1. SalesNote shows an error message.
+    * 2b1. SalesNote shows an error message.
+
+      Use case resumes from step 3.
+
+
+#### Use case: Edit a client
```

Comment:
Is it worth mentioning the case where the client's name is modified?

Reply:
Thanks, I've now included the case